api/src/models/bcmi/collection.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+module.exports = require('../../utils/model-schema-generator')(
+  'BCMICollection',
```

Comment:
Is there any desire/future potential for collections to be needed by any other system? I doubt it, but if so maybe DocCollection or something rather than specifying BCMI?

Also, really picky, but we use MineBCMI for the Mines, maybe this should be CollectionBCMI just for consistency in the naming style?

Reply:
Snap - good catch.

Reply:
Fixed in 7511fa3